clerk/session_claims_test.go

```diff
@@ -0,0 +1,27 @@
+package clerk
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSessionClaims_HasPermissiont(t *testing.T) {
+	// user has permission
+	hasPermission := dummySessionClaims.HasPermission("org:billing:manage")
+	assert.True(t, hasPermission)
+
```

Comment:
🔧 You could also assert that the helper finds both permissions defined in the claims:

https://github.com/clerk/clerk-sdk-go/blob/fede4e5118f91993dce476862637c600a376a533/clerk/tokens_test.go#L56

This case may provide an additional guarantee that we perform an exhaustive search.

Reply:
Will add! Thanks for the suggestion